### PR TITLE
Get api 4

### PR DESCRIPTION
**Pull request by tashkenna**

### CODE REVIEW

File: APP/app.test.js [hunk 1]
```diff
@@ -34,14 +34,33 @@ describe("GET /api/topics", () => {
...
+      .get("/api/")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(descriptions);
+      });
   });
-  
+});
 
+describe("GET /api/articles", () => {
```
Comment (Hard-Luck) [inline APP/app.test.js]: I would add the. :id to the end of this description to show it's a parametric endpoint

**Review by Hard-Luck — CHANGES_REQUESTED** [review]
What you have is great, you need to consider what errors could occur. 
How could the user mess up a request etc...

File: APP/app.js [hunk 2]
```diff
@@ -1,12 +1,28 @@
 const express = require("express");
-const { getTopics, getEndpoints } = require("./controller");
+const { getTopics, getEndpoints, getArticles } = require("./controller");
 const app = express();
-const descriptions = require('../endpoints.json')
+const descriptions = require("../endpoints.json");
+
+app.get("/api/", (req, res) => {
+  res.status(200).send(descriptions);
+});
```
Comment (yalicee) [inline APP/app.js:8]: Would like this to be within a controller which sends on the key of endpoints

File: APP/app.js [hunk 3]
```diff
@@ -1,12 +1,28 @@
...
 
 app.get("/api/topics", getTopics);
 
-app.get("/api/", (req, res) => {
-    res.status(200).send(descriptions)
```
Comment (yalicee) [inline APP/app.js:9]: change the function to getArticleById

File: APP/model.js [hunk 4]
```diff
@@ -16,3 +16,13 @@ FROM
     });
 };
 
+exports.selectArticles = (id) => {
```
Comment (yalicee) [inline APP/model.js]: selectarticlebyId is better name

**Pull request merged**